NorthstarDLL/squirrelnativeautobind.h

```diff
@@ -0,0 +1,97 @@
+#pragma once
```

Comment:
name of this file could probably be a bit shorter, just squirrelautobind would work